src/lib/server/models.ts

```diff
@@ -84,11 +73,65 @@ const modelConfig = z.object({
 
 const modelsRaw = z.array(modelConfig).parse(JSON5.parse(MODELS));
 
+async function getChatPromptRender(
```

Comment:
this is singleton right? This only gets called once at init. Is this correct assumption?

Reply:
yes correct! it's called where the template compilation used to take place